WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift

```diff
@@ -0,0 +1,256 @@
+
+                HStack(spacing: Layout.contentPadding) {
+                    // TODO: dynamic content
+                    // Description
+                    Text("Get the latest white shirt for a stylish look")
+                        .fontWeight(.semibold)
+                        .headlineStyle()
+                        .multilineTextAlignment(.leading)
+
+                    // Simulate shop button
+                    Text(Localization.shopNow)
+                        .fontWeight(.semibold)
+                        .captionStyle()
+                        .padding(Layout.contentMargin)
+                        .background(Color(uiColor: .systemGray6))
+                        .cornerRadius(Layout.adButtonCornerRadius)
+                }
+            }
+            .padding(Layout.contentPadding)
+            .background(Color(uiColor: .systemBackground))
+            .cornerRadius(Layout.cornerRadius)
+            .shadow(color: .black.opacity(0.05),
+                    radius: Layout.shadowRadius,
+                    x: 0,
+                    y: Layout.shadowYOffset)
+
+            // Button to edit ad details
+            Button(action: {
+                // TODO
+            }, label: {
+                Text(Localization.editAd)
+                    .fontWeight(.semibold)
+                    .font(.body)
+                    .foregroundColor(.accentColor)
+            })
+            .buttonStyle(.plain)
+        }
+        .padding(Layout.contentPadding)
+        .background(Color(uiColor: .systemGray6))
+        .cornerRadius(Layout.cornerRadius)
+        .padding(.vertical, Layout.contentPadding)
+    }
+
+    func detailView(title: String, content: String, action: @escaping () -> Void) -> some View {
+        Button(action: action, label: {
+            HStack {
+                VStack(alignment: .leading, spacing: Layout.detailContentSpacing) {
+                    Text(title)
+                        .bodyStyle()
+                    Text(content)
+                        .secondaryBodyStyle()
+                        .multilineTextAlignment(.leading)
+                }
+                Spacer()
+                Image(systemName: "chevron.right")
+                    .secondaryBodyStyle()
+            }
+            .padding(.horizontal, Layout.contentPadding)
+            .padding(.vertical, Layout.contentMargin)
+        })
+    }
+
+    var divider: some View {
+        Divider()
+            .frame(height: Layout.strokeWidth)
+            .foregroundColor(Color(uiColor: .separator))
+    }
+
+    var roundedRectangleBorder: some View {
+        RoundedRectangle(cornerRadius: Layout.cornerRadius)
+            .stroke(Color(uiColor: .separator), lineWidth: Layout.strokeWidth)
+    }
+}
+
+private extension BlazeCampaignCreationForm {
+    enum Layout {
+        static let contentMargin: CGFloat = 8
+        static let contentPadding: CGFloat = 16
+        static let imagePadding: CGFloat = 8
```

Comment:
nit: This isn't used now.

Reply:
Removed in 8f4b4f1ee9ba29f017ae4d0ef63364f4eb03b456.